tests/test_transform.py

```diff
@@ -1,6 +1,6 @@
 """
 Description:
-This script does unit test for transform.py. It includes three tests:
+This script does units test for transform.py. It includes three tests:
```

Comment:
This is not a serious nit, but should this be "unit tests" rather than "units test"?

Reply:
Yes it should, which makes me think I missed something else that should have gone from unit to units.

Reply:
22bac53. No I just put the s in the wrong place.

Reply:
Cool. Thanks for the update.